icu4c/source/test/depstest/dependencies.txt

```diff
@@ -987,16 +990,41 @@ group: numberformatter
     number_decimfmtprops.o
     number_fluent.o number_formatimpl.o
     number_grouping.o number_integerwidth.o number_longnames.o
-    number_mapper.o number_modifiers.o number_multiplier.o
+    number_mapper.o number_modifiers.o
     number_notation.o number_padding.o
-    number_patternmodifier.o number_patternstring.o number_rounding.o
+    number_patternmodifier.o number_patternstring.o
     number_scientific.o
-    currpinf.o dcfmtsym.o numsys.o
+    currpinf.o
     numrange_fluent.o numrange_impl.o
   deps
-    decnumber double_conversion formattable units
-    number_representation number_output
+    decnumber double_conversion formattable units unitsformatter
+    listformatter number_representation number_output
+    numsys
+    number_usageprefs
     uclean_i18n common
+    number_symbolswrapper
```

Comment:
I think numberformatter should depend directly on number_rounding?

Reply:
Done.